lib/hyku_addons/engine.rb

```diff
@@ -230,6 +230,7 @@ def self.dynamically_include_mixins
       User.include HykuAddons::UserEmailFormat
       Bolognese::Writers::RisWriter.include Bolognese::Writers::RisWriterBehavior
       Hyrax::GenericWorksController.include HykuAddons::WorksControllerBehavior
+      Hyrax::DOI::HyraxDOIController.include HykuAddons::DOIControllerBehavior
```

Comment:
If we come up with a new pattern that we like and is generic, I think it would be good to push it back upstream to `hyrax-doi` and hopefully remove this override eventually.